openslides/agenda/projector.py

```diff
@@ -69,31 +69,63 @@ def get_collection_elements_required_for_this(self, collection_element, config_e
             output.extend(self.get_requirements_as_collection_elements(config_entry))
         return output
 
+    def update_data(self):
```

Comment:
You can just return config_entry.get('id'). This is the same as the pk of the item. I think you want to get the pk and not the human readable item_number.